projects/angular/src/forms/combobox/_properties.combobox.scss

```diff
@@ -4,36 +4,38 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 @use '../../utils/mixins';
-@use '../../utils/variables/variables';
-@use 'variables.combobox' as combobox-variables;
-@use '../../popover/dropdown/variables.dropdown' as dropdown-variables;
-@use '../styles/variables.forms' as forms-variables;
+@use '@cds/core/tokens/tokens.scss';
 
 @include mixins.exports('combobox.properties') {
-  @if variables.$clr-use-custom-properties == true {
-    :root {
+  :root {
+    [cds-theme] {
+      --clr-combobox-border-color: #{tokens.$cds-alias-object-border-color};
       --clr-combobox-trigger-color: var(--clr-color-neutral-500);
```

Comment:
Is `clr-combobox-trigger-color` used anywhere? I don't see any references, probably is safe for removal.

Reply:
Good catch, removing